docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -488,10 +445,41 @@ class Sidebar extends React.Component<any, any> {
           />
           <Text color="white" content={pkg.version} size="medium" styles={logoStyles} />
         </Segment>
+        <Flex column>
+          <a
+            href={constants.repoURL}
+            target="_blank"
+            rel="noopener noreferrer"
+            style={topItemTheme}
+          >
+            <Box>
+              GitHub
+              <Icon name="github" styles={{ float: 'right' }} />
+            </Box>
+          </a>
+          <a href={changeLogUrl} target="_blank" rel="noopener noreferrer" style={topItemTheme}>
+            <Box>
+              CHANGELOG
+              <Icon name="file alternate outline" styles={{ float: 'right' }} />
+            </Box>
+          </a>
+          <Input
+            styles={topItemTheme}
+            fluid
+            icon={{ name: 'search', style: { padding: topItemTheme.padding } }}
+            placeholder="Search"
+            iconPosition="start"
+            role="search"
+            onChange={this.handleQueryChange}
+            value={this.state.query}
+            ref={this.searchInputRef}
```

Comment:
```suggestion
            inputRef={this.searchInputRef}
```

We have `inputRef`, so we can simplify it